### PR TITLE
added history functionality

### PR DESCRIPTION
**Pull request by nikhiljugale007**
Files to check:
History.jsx
VideoCard.jsx
apicalls.js

### CODE REVIEW

**Review by sri-rishi — APPROVED** [review]
Great work, couldn't find any issue, feel free to merge

File: src/components/video-card/VideoCard.jsx [hunk 1]
```diff
@@ -12,12 +12,16 @@ import {
...
 	removeFromLikedVideos,
+	addToHistory,
+	removeFromHistory,
 } from "../../api/apicalls";
-import { Link } from "react-router-dom";
+import { useNavigate } from "react-router-dom";
 import { useVideoContext } from "../../context/VideoContext";
-const VideoCard = ({ video }) => {
+const VideoCard = ({ video, card_type }) => {
+	console.log(card_type);
```
Comment (Sahil25061999) [inline src/components/video-card/VideoCard.jsx]: remove console if not needed.

**Review by Sahil25061999 — APPROVED** [review]
the history functionality is working fine, the code structure is good, suggested a minor change.

File: src/components/video-card/VideoCard.jsx (hunk 1 above)
Comment (nikhiljugale007) [reply src/components/video-card/VideoCard.jsx]: removed.

**Pull request merged**